uber/decorators.py

```diff
@@ -500,6 +500,6 @@ def check_id(self, **params):
             else:
                 message = "You weren't found in our database."
                 if session.query(sa.Attendee).filter(sa.Attendee.id == params['id']).first():
-                    return func(**params)
+                    return func(*args, **params)
```

Comment:
The reason why it wasn't worked before when you had ``self`` is that you weren't actually passing ``self`` to this function, i.e. you would have needed to say

```
return func(self, **params)
```